engine/resources/resources_test.go

```diff
@@ -195,6 +215,15 @@ func ErrIsNotExistOK(e error) error {
 	return errwrap.Wrapf(e, "unexpected error")
 }
 
+// GetUID returns the UID of the running user.
```

Comment:
Is this what you want?

https://github.com/purpleidea/mgmt/blob/be39fbeff6812e35454f0381f9ad551ac4f776c2/engine/util/util.go#L332

Reply:
This function gets the uid from the user defined in the `$USER` environment variable. I updated the comment, let me know if it makes more sense !